vowpalwabbit/json_parser/tests/dsjson_parser_test.cc

```diff
@@ -236,9 +230,8 @@ TEST(ParseDsjson, Cats)
   }
 }
 )";
-  auto vw = VW::initialize(
-      "--dsjson --chain_hash --cats 4 --min_value=185 --max_value=23959 --bandwidth 1 --no_stdin --quiet", nullptr,
-      false, nullptr, nullptr);
+  auto vw = VW::initialize(vwtest::make_args("--dsjson", "--chain_hash", "--cats", "4", "--min_value=185",
+      "--max_value=23959", "--bandwidth", "1", "--no_stdin", "--quiet"));

```

Comment:
need two strings for cmds with '=' ?

Reply:
No, `--max_value=23959` is a valid single token. In fact it is preferred as it is unambiguous to the CLI parser.